index.html

```diff
@@ -278,6 +278,75 @@ <h3>
           reset because its [=permission/lifetime=] has expired.
         </p>
       </section>
+      <section>
+        <h3>
+          Permission Store
+        </h3>
+        <p>
+          The user agent maintains a single <dfn class="export">permission store</dfn> which is a [=/list=] of [=permission store entries=].
+        </p>
+        <p>
+          The user agent MAY remove [=entries=] from the [=permission store=] when their respective [=permission=]'s [=permission/lifetime=] has expired.
+        </p>
+        <p>
+          A <dfn class="export" data-local-lt="entry">permission store entry</dfn> is a [=tuple=] of {{PermissionDescriptor}} <dfn class="export" data-dfn-for="permission store entry">descriptor</dfn>, [=permission key=] <dfn class="export" data-dfn-for="permission store entry">key</dfn>, and [=permission/state=] <dfn class="export" data-dfn-for="permission store entry">state</dfn>.
+        </p>
+        <p>
+        To <dfn class="export">get a permission store entry</dfn> given a {{PermissionDescriptor}} |descriptor| and [=permission key=] |key|, run these steps:
+          <ol class="algorithm">
+            <li>
+              If the user agent's [=permission store=] [=list/contains=] an [=entry=] whose [=permission store entry/descriptor=] is |descriptor|, and whose [=permission store entry/key=] [=permission key/is equal to=] |key| given |descriptor|, return that entry.
+            </li>
+            <li>
+              Return null.
```

Comment:
Question here, and generally: do you envision the permission store "bootstrapped" with entries for each supported feature w/ the default permission state ("prompt")? Or do you remove the entry when state is removed/ungranted? If the former, we would only return null if the descriptor/key was nonsense, right?

Reply:
Hmm, that's an interesting question. I think returning null here is really to be explicit that there's no entry, and I figured that defaulting to "prompt" (or something else) would be done when computing the permission state when there's no explicit entry in the store (e.g. in step 8 [here](https://pr-preview.s3.amazonaws.com/johannhof/permissions/pull/390.html#dfn-permission-state))

Reply:
Ah, yeah - makes sense. Thx.